runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ModelCoders.java

```diff
@@ -59,6 +60,14 @@ private ModelCoders() {}
 
   public static final String ROW_CODER_URN = getUrn(StandardCoders.Enum.ROW);
 
+  public static final String STATE_BACKED_ITERABLE_CODER_URN =
+      "beam:coder:state_backed_iterable:v1";
```

Comment:
beam_runner_api.proto already have this constant, aren't we supposed to get this constant string from proto definition?

Reply:
We use a string constant here so that it can be used in switch/case statements. We make sure that the string constant matches the enum in the proto using the static block below.